src/benchmarks/clo.c

```diff
@@ -38,14 +38,19 @@
 #include <errno.h>
 #include <assert.h>
 #include <sys/queue.h>
+#include <inttypes.h>
 
 #include "benchmark.h"
 #include "scenario.h"
 #include "clo_vec.h"
 #include "clo.h"
 
+#ifndef min
```

Comment:
This is lazy coding. Fix it properly please. I suggest using std::min, std::max.